example/app.js

```diff
@@ -336,6 +336,44 @@ class App extends React.Component {
           </div>
 
         </div>
+
+        <div className='app-sandbox'>
```

Comment:
Normally, this stuff goes into the documentations. In this case, we're trying to fix a bug so it is not necessary to update the example.

Reply:
Using multiple galleries on a page is a custom use case and most users who download this will probably not leverage multiple galleries on one page. On the app page, Id like to only showcase features that majority of the user will use/ or will find useful.

Reply:
FTR: resolved for now and added to tasks above.